src/patternfly/components/OptionsMenu/options-menu.scss

```diff
@@ -174,9 +174,9 @@
   &:disabled {
     pointer-events: none;
 
-    &:not(.pf-m-plain) {
```

Comment:
Removing this gives a background to the plain (non `.pf-m-text`) button.

<img width="186" alt="Screen Shot 2019-07-09 at 10 32 16 AM" src="https://user-images.githubusercontent.com/35148959/60902272-fa215b80-a234-11e9-82a2-8dfeb933947e.png">

I also don't know that you need the `&:not(.pf-m-plain)` on L179, either. It's correct code-wise, but seems unnecessary and increases specificity.

What if you rewrote this block to


```
  &.pf-m-disabled,
  &:disabled {
    pointer-events: none;

    &:not(.pf-m-plain),
    &.pf-m-text {
      --pf-c-options-menu__toggle--Background: var(--pf-c-options-menu__toggle--disabled--BackgroundColor);
    }

    &::before {
      border: 0;
    }
  }
```